networkit/centrality.pyx

```diff
@@ -1917,15 +1917,15 @@ def relativeRankErrors(rx, ry):
 		$$r_x(u) / r_y(u)$$
 
 
-	Parameters
-	----------
+	Parameters:
+	-----------
 	rx : list
 		ranking - ordered list of tuples (node, score)
 
 	ry:  list
 		ranking - ordered list of tuples (node, score)
 
-	Returns
+	Returns:
 	_______
```

Comment:
Previously (e.g L 1896) you replaced a line with dashes, should that be the case here as well?